Lunch/LunchPFTableViewController.swift

```diff
@@ -0,0 +1,138 @@
+        self.pullToRefreshEnabled = true
+        self.paginationEnabled = false
+    }
+
+    override func viewDidLoad() {
+        self.navigationItem.hidesBackButton = true
+    }
+
+    override func queryForTable() -> PFQuery {
+        let query = PFQuery(className: classKey)
+
+        query.orderByDescending(voteKey)
+        return query
+    }
+
+    override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath, object: PFObject?) -> PFTableViewCell? {
+        var cell = tableView.dequeueReusableCellWithIdentifier("lunchPlaceCell") as! PFTableViewCell!
+        if cell == nil {
+            cell = PFTableViewCell(style: .Subtitle, reuseIdentifier: "lunchPlaceCell")
+        }
+        if let placeName = object?[placeKey] as? String {
+            cell?.textLabel?.text = placeName
+        }
+
+        if let voteCount = object?[voteKey] as? Int {
+            cell?.detailTextLabel?.text = "Votes: \(voteCount)"
+        }
+
+        return cell
+    }
+
+    override func tableView(tableView: UITableView, editActionsForRowAtIndexPath indexPath: NSIndexPath) -> [AnyObject]? {
+        let delete = UITableViewRowAction(style: .Normal, title: "Delete") { action, index in
+            self.objectAtIndexPath(indexPath)?.deleteInBackgroundWithBlock({ (success: Bool, error: NSError?) -> Void in
+                if success && error == nil {
+                    self.loadObjects()
+                    self.tableView.reloadData()
+                }
+            })
+
+        }
+        delete.backgroundColor = UIColor.redColor()
+
+        // VOTE
+        let vote = UITableViewRowAction(style: .Normal, title: "Vote") { action, index in
+            var voteCount = self.objects?[indexPath.row][self.voteKey] as! Int
+            self.objectAtIndexPath(indexPath)?.incrementKey("vote")
+            self.objectAtIndexPath(indexPath)?.saveInBackgroundWithBlock({ (success: Bool, error: NSError?) in
+                if success && error == nil {
+                    self.loadObjects()
+                    self.tableView.reloadData()
+                }
+            })
+        }
+        vote.backgroundColor = UIColor.blueColor()
+        return [vote, delete]
+    }
+
+    override func tableView(tableView: UITableView, canEditRowAtIndexPath indexPath: NSIndexPath) -> Bool {
+        // the cells you would like the actions to appear needs to be editable
+        return true
+    }
+
+    override func tableView(tableView: UITableView, commitEditingStyle editingStyle: UITableViewCellEditingStyle, forRowAtIndexPath indexPath: NSIndexPath) {
+        // you need to implement this method too or you can't swipe to display the actions
+    }
+
+    @IBAction func addPlaceBarItemTapped(sender: UIBarButtonItem) {
+        var inputTextField: UITextField?
+        let eateryPrompt = UIAlertController(title: "Enter an Eatery", message: "Look up the best place to eat!", preferredStyle: UIAlertControllerStyle.Alert)
```

Comment:
So you're using a `UIAlertController` here but `UIAlertView` elsewhere… I'd recommend sticking with just one, and probably the controller because UIAlertView is deprecated :stuck_out_tongue: